meta-aos-rpi-domd/recipes-core/images/aos-image-rpi.bb

```diff
@@ -0,0 +1,69 @@
+SUMMARY = "Aos image for Raspberry Pi devices"
+
+require recipes-core/images/core-image-minimal.bb
+require recipes-core/images/aos-image.inc
+
+# Enable package manager
+EXTRA_IMAGE_FEATURES += "package-management"
+
+RDEPENDS += "rpi-bootfiles trusted-firmware-a"
+
+do_image[depends] += " \
+    rpi-bootfiles:do_deploy \
+    trusted-firmware-a:do_deploy \
+    ${@bb.utils.contains('RPI_USE_U_BOOT', '1', 'u-boot-tools-native:do_populate_sysroot', '',d)} \
+    ${@bb.utils.contains('RPI_USE_U_BOOT', '1', 'u-boot:do_deploy', '',d)} \
+    ${@bb.utils.contains('RPI_USE_U_BOOT', '1', 'u-boot-default-script:do_deploy', '',d)} \
+"
+
+PACKAGE_INSTALL:append = " \
```

Comment:
Unclear why it is needed?

Reply:
Referred to xen team example.
Replaced with IMAGE_INSTALL, works fine.